feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -235,20 +238,20 @@ func (tc *testCase) verifyInterfaceDUT(
 			}
 		}
 	}
-
-	disp := dip.Subinterface(0)
-
-	// IPv4 neighbor discovered by ARP.
-	dis4np := disp.Ipv4().Neighbor(atea.IPv4)
-	if got := dis4np.Origin().Get(t); got != dynamic {
-		t.Errorf("%s IPv4 neighbor %s origin got %v, want %v", dp, atea.IPv4, got, dynamic)
+	if *deviations.MissingSubinterfaceOriginLeaf {
```

Comment:
Deviation must be designed so that "false" is OC compliant, and "true" is deviated behavior. This is the opposite and not acceptable.

Reply:
@liulk , Changed it